crib/scripts/check_env_vars.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# List of required environment variables
+required_vars=(
+  "DEVSPACE_IMAGE"
+  "DEVSPACE_INGRESS_CIDRS"
+  "DEVSPACE_INGRESS_BASE_DOMAIN"
+  "DEVSPACE_INGRESS_CERT_ARN"
+  "DEVSPACE_K8S_POD_WAIT_TIMEOUT"
+  "NS_TTL"
+  )
+
+missing_vars=0  # Counter for missing variables
+
+# Check each variable
+for var in "${required_vars[@]}"; do
+  if [ -z "${!var}" ]; then  # If variable is unset or empty
+    echo "Error: Environment variable $var is not set."
+    missing_vars=$((missing_vars + 1))
+  fi
+done
+
+# Exit with an error if any variables were missing
+if [ $missing_vars -ne 0 ]; then
+  echo "Total missing environment variables: $missing_vars"
```

Comment:
It might not be obvious to someone what vars they need to set from this message. WDYT about printing the `$required_vars` here or printing the path to the `.env.example` file?

Reply:
good point, I updated the script to print more instructions